client/my-sites/earn/main.tsx

```diff
@@ -24,7 +24,7 @@ import ReferAFriendSection from './refer-a-friend';
 import { Query } from './types';
 
 type EarningsMainProps = {
-	section: string;
+	section?: string;
```

Comment:
While updating Ad methods, I realized this param is optional and is sometime undefined. Needed to update it here for handling in some of the methods below.